src/MixedModels.jl

```diff
@@ -50,6 +53,8 @@ export @formula,
     LogLink,
     MixedModel,
     MixedModelBootstrap,
+    MixedModelProfile,
+    Not,
```

Comment:
why export `Not`?

Reply:
I'm "Not" sure why I did that.  I'll revert it.